src/main/java/com/ldtteam/structurize/management/StructureName.java

```diff
@@ -17,6 +21,9 @@
 {
     public static List<String> HUTS = Collections.synchronizedList(new ArrayList<>());
 
+    private static Map<String, Component> SECTIONS = createDefaultSections();
```

Comment:
What is all of this for?

Reply:
Custom build sections.  See examples in the Minecolonies PR.